mlflow/__init__.py

```diff
@@ -153,9 +156,16 @@
         xgboost,
     )
 
-if MLFLOW_CONFIGURE_LOGGING.get() is True:
+if os.environ.get("MLFLOW_LOGGING_CONFIGURE_LOGGING", "true").lower() == "false":
+    warnings.warn(
+        "MLFLOW_LOGGING_CONFIGURE_LOGGING is deprecated and will be removed in a future release, "
+        f"please use {MLFLOW_CONFIGURE_LOGGING.name} instead.",
+        FutureWarning,
+    )
+elif MLFLOW_CONFIGURE_LOGGING.get() is True:
```

Comment:
I think this logic should put in `_BooleanEnvironmentVariable` or `_EnvironmentVariable`.

Reply:
For example, we could add an option like `deprecated` that takes a list of arguments and put aliases there.

Reply:
I don't think this is common though, this issue should be a typo introduced at the initial version

Reply:
I see, then let’s update the `get_raw` method.